node6

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env node
+
+// Copyright 2016 Traceur Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+// The System object require()s traceur. The module sets global.System
+// as a side-effect.
+require('./src/node/System.js');
+
+// The first argument is 'node6'; the second one is our 'program' name.
+var filename = process.argv[2];
+
+// Interpret the filename argument as a platform-independent,
+// normalized module name.
+var moduleName = filename.replace(/\\/g, '/');
+
+// Simuulate a future where it looks like node is executing our program
+process.argv = ['node'].concat(process.argv.slice(2));
```

Comment:
I would do `node6` instead of `node` here. Or maybe read from `process.execPath`, since people might have invoked it with any variety of (node|iojs)[.exe]